geode-core/src/main/java/org/apache/geode/internal/cache/PartitionedRegionDataStore.java

```diff
@@ -2953,7 +2953,7 @@ public void executeOnDataStore(final Set localKeys, final Function function, fin
     long start = stats.startFunctionExecution(function.hasResult());
     try {
       if (logger.isDebugEnabled()) {
-        logger.debug("Executing Function: {} on Remote Node with context: ", function.getId(),
+        logger.debug("Executing Function:{} on Remote Node with context: {}", function.getId(),
```

Comment:
The space was removed in `Function:{}`. It probably should also be removed in `context: {}`